### PR TITLE
feat: on Build Details Page replace tests table by tests table from Tests tab

### PR DESCRIPTION
**Pull request by lfjnascimento**
- make TestTable more flexible to reuse it on other pages
- use TestTable in BuildDetails page

Close: #386 

### How to tests:
Check out the `Test Results` section of a BuildDetails Page

| Before | After |
|--------|--------|
| image | image |

### CODE REVIEW

**Comment by WilsonNet** [comment]
please add a ticket for adding the hardware to this table like in 
#408

**Comment by lfjnascimento** [comment]
> please add a ticket for adding the hardware to this table like in #408
Since it is the same table I will just complement the #408

File: backend/kernelCI_app/views/buildTestsView.py [hunk 1]
```diff
@@ -1,48 +1,25 @@
...
-from django.db.models.functions import Concat
-from django.contrib.postgres.aggregates import ArrayAgg
 
 from kernelCI_app.models import Tests
-from kernelCI_app.serializers import BuildTestsSerializer
 
 
 class BuildTests(View):
 
     def get(self, request, build_id):
```
Comment (MarceloRobert) [inline backend/kernelCI_app/views/buildTestsView.py:9]: Is the `request` param being used?

**Comment by lfjnascimento** [comment]
> please add a ticket for adding the hardware to this table like in #408

created https://github.com/orgs/kernelci/projects/28/views/3?pane=issue&itemId=84865243

**Review by anajalvarenga — APPROVED** [review]

File: dashboard/src/types/general.ts [hunk 2]
```diff
@@ -24,23 +24,18 @@ export type TIndividualTest = {
...
-};
-
 export type TIssue = {
   id: string;
   comment?: string;
   report_url?: string;
   incidents_info: IncidentsInfo;
 };
+
+export type TestHistory = {
```
Comment (WilsonNet) [inline dashboard/src/types/general.ts:34]: Nit: we alredy have this one being defined in another type file

File: backend/kernelCI_app/views/buildTestsView.py (hunk 1 above)
Comment (lfjnascimento) [reply backend/kernelCI_app/views/buildTestsView.py:9]: No, but I am forced to accept it in order to also accept the `build_id`, as it is the third argument of the function.

**Review by MarceloRobert — APPROVED** [review]

**Pull request merged**